app/main.py

```diff
@@ -1 +1,62 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        skills = ([
```

Comment:
There is no need to define the `skills` variable, extend the `self.skills` in one operation. Make this change in all classes